packages/fx-core/src/common/armInterface.ts

```diff
@@ -37,3 +37,7 @@ export interface ScaffoldArmTemplateResult extends Record<string, unknown> {
   Modules?: { [moduleFileName: string]: BicepModule };
   Orchestration: BicepOrchestration;
 }
+
+export interface BicepPluginsContext {
```

Comment:
This interface is to keep the keys like in bicep script the same with the  handlebars context. It could be better for all plugins' bicep script keeping the same key style.